productImport/product/productHandler.go

```diff
@@ -9,15 +9,15 @@ import (
 type ProductHandler struct {
 	fileManager *adapters.FileManager
 	handler     adapters.HandlerInterface
-	columnMap *ColumnMap
+	ColumnMap   *ColumnMap
```

Comment:
What the reason to make it public?

Reply:
it used in report tests now.

will be removed after product's handler data init refactoring